src/v2/view-builder/views/email/RequiredFactorEmailView.js

```diff
@@ -6,7 +6,7 @@ import polling from '../shared/polling';
 import BaseFactorView from '../shared/BaseFactorView';
 import { addSwitchAuthenticatorLink } from '../../utils/AuthenticatorUtil';
 
-const SHOW_RESEND_TIMEOUT = 60000;
+const SHOW_RESEND_TIMEOUT = 30000;
```

Comment:
testcafe test `ChallengeFactorEmail_MagicLink_spec.js` need update as well.

Reply:
updated the test to have less timeout...